deploy/manifests/sample-deployments/kong-enterprise-hybrid/kong-cp.env

```diff
@@ -0,0 +1,3 @@
+KONG_ADMIN_API_URI=http://192.168.1.100:30801
+KONG_ADMIN_GUI_URL=http://192.168.1.100:30802
```

Comment:
These rely on specific port/network configuration that won't be present outside any one environment, and shouldn't be provided in specific examples as such. https://docs.konghq.com/enterprise/2.1.x/deployment/installation/docker/#step-6-start-kong-enterprise-with-kong-manager-and-kong-developer-portal-enabled and https://github.com/Kong/charts/blob/main/charts/kong/README.md#service-location-hints are examples of guidance we provide elsewhere.

Reply:
We did this example to "show" which values one need to change to use the kustomize bases, like: "you can leave everything else default BUT remember to patch these values".

Do you have some suggestion to achieve that?